packages/ui-components/src/types.ts

```diff
@@ -35,3 +35,4 @@ export * from './components/toggle-tip/toggle-tip.types';
 export * from './components/tooltip/tooltip.types';
 export * from './components/tag-status/tag-status.types';
 export * from './components/tag-alarm/tag-alarm.types';
+export * from './components/wizard-header/wizard-header.types';
```

Comment:
Don't forget to add the angular component.

Reply:
Adding a wizard here is indeed the plan :P I have to make some changes to this component due to the redesign.
Thanks for pointing out the lack of export on the angular lib, totally missed that.